app/modules/ads/ml/tb-viewability.js

```diff
@@ -0,0 +1,73 @@
+let config = {};
+
+export const tbViewability = {
+  configureTbViewability(billTheLizardConfig) {
+    const {
+      context,
+      ViewabilityCounter,
+    } = window.Wikia.adEngine;
+    const { billTheLizard } = window.Wikia.adServices;
```

Comment:
You should be able to unpack all these values from `window.Wikia.adEngine`.

Reply:
👍